### ISSUE TITLE
In the ER diagram in the related tables, the show mode is synchronized with the main pane

### ISSUE THREAD

**Issue by MH4GF** [liam-hq/liam#334]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](../discussions/categories/q-a).
- [X] I have searched for existing issues [search for existing issues](../issues), including closed ones.

### Cloud or Self Hosted

Self Hosted

### Version (only for Self Hosted)

v0.0.12 + f4c6be3 (2024-12-20)

### Steps to reproduce

https://github.com/user-attachments/assets/6a1f559f-e842-4a21-b68e-8b77fda12ecb

### Expected Behavior

In the ER diagram in the related tables, the show mode should be table name only.

### Actual Behavior

The show mode is synchronized with the main pane.

**Comment by Jal-GG** [comment]
hey can i help with the bug ? as this is my first open source contribution can you guide me also?

**Comment by MH4GF** [comment]
Hey @Jal-GG, We welcome contributions!
I would like you to read https://github.com/liam-hq/liam/blob/main/CONTRIBUTING.md.
Are you interested in working on this issue?

**Comment by Jal-GG** [comment]
@MH4GF yes very much excited for my first contribution but i am a newbie to dev so ... 😃

**Comment by MH4GF** [comment]
@Jal-GG In the meantime, try to implement it! Let me know if you need help.

**Comment by Jal-GG** [comment]
hey @MH4GF  this package.json has regex in the build command also "CP" is used which is not running properly in windows , i have been figuring out for a day to install the dependency 😆 . you might me using mac or linux i guess.

**Comment by Jal-GG** [comment]
hey @MH4GF this all the configs and package scripts are for unix or mac compatible, so.. not working on windows :)

**Comment by MH4GF** [comment]
@Jal-GG 
Nice. I don't have a Windows device so it's hard to test, but by all means make an issue!

**Comment by Jal-GG** [comment]
@MH4GF hey can we make this project containerized so no more building errors?

**Comment by MH4GF** [comment]
@Jal-GG I think it is good idea. If you're interested, give it a try!

**Comment by Jal-GG** [comment]
@MH4GF i am on it brother sorry for the late response
